### PR TITLE
Update SSWG members list

### PR DESCRIPTION
**Pull request by 0xTim**

### CODE REVIEW

**Review by adam-fowler — APPROVED** [review]
LGTM

**Review by ktoso — APPROVED** [review]
LGTM :)

File: README.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ Analogous to the Core Team for Swift, the work group is responsible for providin
 
 The current Swift Server work group consists of the following people:
 
-* Gwynne Raskind ([@gwynne](https://github.com/gwynne), Vapor)
```
Comment (ktoso) [inline README.md:13]: I don't think we want to remove Gwynne right?

Comment (0xTim) [reply README.md:13]: Updated

**Review by ktoso — APPROVED** [review]

**Comment by ktoso** [comment]
Thanks!

**Pull request merged**